scripts/docker/data/conf.py

```diff
@@ -0,0 +1,328 @@
+# -*- coding: utf-8 -*-
+#
+# Forseti Security documentation build configuration file, created by
+# sphinx-quickstart on Tue Apr  3 22:56:04 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+
+sys.path.insert(0, os.path.abspath('.'))
```

Comment:
@denzel-morris How do we maintain the inserts going forward w/o having to remember to do it? Can we wildcard this here or use a python method (os.walkdir)?